Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:21.04
```

Comment:
This file should not be such prominent in the root directory of the project, I think, people might think that this builds the project output. Here is my proposal:

- move it to a sub-directory inside https://github.com/shipwright-io/build/tree/main/images
- we can build the image nightly, for example in https://github.com/shipwright-io/build/blob/main/.github/workflows/base-images.yaml, the file name is a bit mis-leading (could be images.yaml instead), I think you can even let it build multi-platform like we do for the git base image

Subsequently, we don't need `make container-image` and `make container-make` can use the image from our shipwright-org.